### DISCUSSION TITLE
ISO9660/LiveCD when?

### DISCUSSION THREAD

**Discussion by ghost** [managarm/managarm#387] [Q&A] [answered]
When will managarm support CD/DVD, because it would be pretty cool to boot the OS off of a CD live.

**Comment by Dennisbonke** [comment, marked as answer]
While we don’t support CD/DVD boot, one can try booting from an USB stick instead.

**Reply by ghost** [reply]
Oof, don't currently have any on hand. Any idea on if/when it will support CD/DVD?

**Reply by ArsenArsen** [reply]
There's no current roadmap for this feature.

**Reply by Dennisbonke** [reply]
However, we would be more than happy to accept a PR implementing this feature in the meantime.
